types/tampermonkey/tampermonkey-tests.ts

```diff
@@ -8,6 +8,9 @@ if (window.onurlchange === null) {
     window.addEventListener('urlchange', ({ url }) => console.log(url));
     // or
     window.addEventListener('urlchange', info => console.log(info.url));
+
+    // window.addEventListener('click', (event) => console.log(e.url));
```

Comment:
Why do we need this?

Reply:
As in the comment on the next line, the comment here is meant to say that this is incorrect and can be identified by types, as only urlchange events return parameters with url objects, see commit [here](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d52402196e295c82ca73c8194d18a57c6d02a02b#r95588217) for details, perhaps the unnecessary comment here could be removed?